components/molecules/m-product-carousel.vue

```diff
@@ -0,0 +1,90 @@
+<template>
+  <SfCarousel class="m-product-carousel">
+    <SfCarouselItem v-for="(product, i) in carouselProducts" :key="i">
+      <SfProductCard
+        :title="product.title"
+        :image="product.image"
+        :regular-price="product.price.regular"
+        :special-price="product.price.special"
+        :max-rating="product.rating.max"
+        :score-rating="product.rating.score"
+        :link="product.link"
+        link-tag="a"
+        :is-on-wishlist="isOnWishlist(product.data)"
+        class="product-card"
+        @click:wishlist="toggleWishlist(product.data)"
+      />
+    </SfCarouselItem>
+  </SfCarousel>
+</template>
+<script>
+import { mapGetters } from "vuex";
+import { SfProductCard, SfCarousel } from "@storefront-ui/vue";
+import { price, htmlDecode } from "@vue-storefront/core/filters";
+import config from "config";
+import { currentStoreView } from "@vue-storefront/core/lib/multistore";
+import { formatProductLink } from "@vue-storefront/core/modules/url/helpers";
+import { productThumbnailPath } from "@vue-storefront/core/helpers";
+export default {
+  name: "MProductCarousel",
+  components: {
+    SfCarousel,
+    SfProductCard
+  },
+  props: {
+    products: {
+      type: Array,
+      default: () => ({})
```

Comment:
Default value for `products` prop should be empty array instead of empty object.

Reply:
done